<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontPage</class>
 <widget class="QMainWindow" name="FrontPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">img url(:/img/images/pamphlet1.jpg)</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="home_1">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>1241</width>
          <height>721</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>562</width>
             <height>132</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/img/images/pushbutton1.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>562</width>
             <height>132</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/img/images/pushbutton3.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>562</width>
             <height>132</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/img/images/pushbutton2.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>562</width>
             <height>132</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/img/images/pushbutton4.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="3">
          <widget class="QLabel" name="logo">
           <property name="minimumSize">
            <size>
             <width>562</width>
             <height>562</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="helpButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>562</width>
             <height>132</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/img/images/pushbutton5.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="home_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QPushButton" name="homeButton_1">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>1250</width>
          <height>760</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Info</string>
         </attribute>
         <widget class="QLabel" name="pamphlet_pic">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1250</width>
            <height>760</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Pricing</string>
         </attribute>
         <widget class="QLabel" name="pricing_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1250</width>
            <height>760</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="home_3">
       <widget class="QPushButton" name="homeButton_2">
        <property name="geometry">
         <rect>
          <x>1070</x>
          <y>0</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
       <widget class="QLabel" name="contact_label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>1250</width>
          <height>760</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="home_4">
       <widget class="QPushButton" name="homeButton_3">
        <property name="geometry">
         <rect>
          <x>992</x>
          <y>10</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reviewButton">
        <property name="geometry">
         <rect>
          <x>992</x>
          <y>40</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Submit A Review</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>reviewButton</sender>
   <signal>clicked()</signal>
   <receiver>FrontPage</receiver>
   <slot>submitReview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1030</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>1129</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpButton</sender>
   <signal>clicked()</signal>
   <receiver>FrontPage</receiver>
   <slot>stopItGetSomeHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>789</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>835</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submitReview()</slot>
  <slot>stopItGetSomeHelp()</slot>
 </slots>
</ui>
